src/backend/src/utils/work-packages.utils.ts

```diff
@@ -26,11 +26,14 @@ export const wpQueryArgs = Prisma.validator<Prisma.Work_PackageArgs>()({
   }
 });
 
+export const calculateWorkPackageProgress = (bullets: Description_Bullet[]) =>
```

Comment:
since this function is called "calculate work package progress", it should take in a work package like object (or, an object `{ expectedActivities: Description_Bullet[]; deliverables: Description_Bullet[] }` and then do the concatenation itself